lnwallet/channel.go

```diff
@@ -2550,6 +2551,7 @@ func (lc *LightningChannel) CompleteCooperativeClose(remoteSig []byte) (*wire.Ms
 	// As the transaction is sane, and the scripts are valid we'll mark the
 	// channel now as closed as the closure transaction should get into the
 	// chain in a timely manner and possibly be re-broadcast by the wallet.
+	lc.MarkChannelAsPendingClose(false)
```

Comment:
The channel shouldn't be marked as closed here as in this context, we don't know if the closure transaction has been broadcast or not.